gnome-initial-setup/pages/branding-welcome/gis-branding-welcome-page.c

```diff
@@ -65,7 +65,7 @@ static void
 read_config_file (GisBrandingWelcomePage *page)
 {
   GisBrandingWelcomePagePrivate *priv = NULL;
-  g_autoptr(GKeyFile) keyfile = NULL;
+  GKeyFile *keyfile = NULL;
```

Comment:
Maybe add the name of the commit to squash this into in the next rebase to the commit message?